ui/page/components/account_selector.go

```diff
@@ -491,3 +489,35 @@ func (asm *AccountSelectorModal) walletInfoPopup(gtx layout.Context) layout.Dime
 		})
 	})
 }
+
+func (asm *AccountSelectorModal) listenForTxNotifications() {
```

Comment:
I wonder if this should be on `accountSelector` which may then update the modal if it is open. Does the selected balance update when a new tx or block comes in?